libuserhelper.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020 Jolla Ltd.
+ * Copyright (c) 2020 Open Mobile Platform LLC.
+ *
+ * All rights reserved.
+ *
+ * BSD 3-Clause License, see LICENSE.
+ */
+
+#ifndef LIBUSERHELPER_H
+#define LIBUSERHELPER_H
+
+#include <QString>
+
+class LibUserHelper
+{
+public:
+    LibUserHelper();
+    uint addGroup(const QString &group);
+    bool removeGroup(const QString &group);
+    bool addUserToGroup(const QString &user, const QString &group);
+    uint addUser(const QString &user, const QString &name, uint guid);
+    bool removeUser(uint uid);
+    bool modifyUser(uint uid, const QString &new_name);
```

Comment:
new_name -> newName

Qtified APIs should use camel-case

Reply:
Renamed